apps/bandwagon/forms.py

```diff
@@ -143,6 +148,15 @@ def __init__(self, *args, **kw):
                 self.instance.type in amo.COLLECTION_SPECIAL_SLUGS):
             del self.fields['slug']
 
+    def clean(self):
+        # Raise the honeypot validation error here to
+        # keep it being rased in the __all__ section of the
```

Comment:
typo `raised`.
Not sure the sentence makes much sense (at least to me), maybe `Check the honeypot here instead of 'clean_your_name' so the error message appears at the top of the form in the __all__ section`?